### ISSUE TITLE
App steals focus on main monitor when opening and closing, even though app opens correctly on second monitor.

### ISSUE THREAD

**Issue by trevor-coleman** [a85/POSTMan-Chrome-Extension#9279]
## Describe the bug**

tl;dr - _When opening the app via spotlight search on my second monitor, the app's loading screen steals focus on the main monitor. And then does so again when closed._

I am using a Macbook with two external monitors attached. in this configuration:

`| Macbook | Monitor 1 | Monitor 2 |`

I use "spaces" (virual desktops) and I typically have my IDE open in fullscreen on Monitor 1 (which is my primary monitor.)

When I use spotlight to open Postman on Monitor 2, monitor 1 switches from my full-screen IDE to the home desktop, and displays the Postman loading screen. Then the app opens on Monitor 2. 

That means I have to swipe back to find my IDE on my main desktop.

When I close the app, once again, the app steals focus on my main monitor and returns it to the home screen, and I have to manually swipe back to my IDE.

It's a minor annoyance, but it really messes with my flow every time it happens.

## To Reproduce 
1. On a MacOS system with multiple monitors and "spaces" virtual desktops enabled, open an app on Monitor1 and use the green button at the top-left to enter full-screen mode.
2. Move the mouse and focus to Monitor2. Press CMD+Space to open spotlight search.
3. Search for Postman, and select the program.
4. **[PROBLEM -->]** Monitor1 will switch away from the program you're using to the primary desktop. 
5. App opens on Monitor2.
5. Manually switch back to the full screen app (CTRL+Left).
6. Focus postman and quit the app via the menu or CMD+Q
7. **[PROBLEM-->]** The Monitor1 will switch away from the program you're using to the main desktop.

## Expected behavior
_Loading and quitting the app from a second monitor should not affect focus on monitors other than the one where the program was launched._

[1-3 - as above]
4. Loading screen shows up on monitor where app was launched.
[5-6 - as above]
7. App closes without affecting focus on other screens.

## Screenshots

These videos show the process on the two monitors -- unfortunately couldn't record both at once, so they won't sync exactly. Also I apparently forgot how to quit apps from the menu -- guess I am too reliant on CMD-Q 😂)

* **Monitor 1:** https://youtu.be/XotQRrOgOqU
* **Monitor 2:** https://youtu.be/NXaNjQ6Vhms

## App information (please complete the following information):
 - App Type [e.g. Chrome App, Native App]
 - Postman Version [e.g. 6.0.1]
 - OS: [e.g. macOS High Sierra 10.13.2]

* Postman: Version 7.34.0 (7.34.0) - Native App
* MacOS: 10.15.7 (19H2)
* Monitor Configuration:
    * https://imgur.com/a/x6EEht2

## Additional context
N/A

**Comment by DannyDainton** [comment]
Hey @trevor-coleman 

Thanks for submitting this - Can I quickly check that this is the same/similar issue? https://github.com/postmanlabs/postman-app-support/issues/6849

**Comment by trevor-coleman** [comment]
In that case it seems that the person is using postman on the same screen, but in a different space.

In my case, it's an issue across multiple monitors.

They're probably caused by the same root issue, but if you're going to merge the issues I think it would be good to make sure that you make a note there about the issue across monitors. That way the fix will works appropriately on multiple monitors, not just multiple spaces on the same monitor.

**Comment by CodePint** [comment]
+1

This still causes me daily annoyances on ubuntu 20.04 LTS
Any ideas or timelines for a fix? Is there any way I can help or contribute to a fix?

**Comment by MC-sanderv** [comment]
This is related to macOS window focus behavior on multi-monitor setups, which is largely controlled by the OS rather than the app as far as I know.
There have been numerous Electron upgrades and improvements to Postman since this issue was reported.

Should the issue still be relevant, please open a new ticket with relevant information.
